src/Gateway/Form/Type/BancontactGatewayConfigurationType.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PayPlug\SyliusPayPlugPlugin\Gateway\Form\Type;
+
+use PayPlug\SyliusPayPlugPlugin\Gateway\BancontactGatewayFactory;
+use PayPlug\SyliusPayPlugPlugin\Gateway\PayPlugGatewayFactory;
+use PayPlug\SyliusPayPlugPlugin\Gateway\Validator\Constraints\IsCanSaveBancontactMethod;
+use PayPlug\SyliusPayPlugPlugin\Gateway\Validator\Constraints\IsCanSaveCards;
+use PayPlug\SyliusPayPlugPlugin\Gateway\Validator\Constraints\IsPayPlugSecretKeyValid;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+final class BancontactGatewayConfigurationType extends AbstractGatewayConfigurationType
+{
+    /**
+     * @inheritdoc
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $validationGroups = ['Default', 'sylius'];
+
+        $builder
+            ->add('secretKey', TextType::class, [
+                'label' => 'payplug_sylius_payplug_plugin.ui.secret_key',
+                'validation_groups' => $validationGroups,
+                'constraints' => [
+                    new NotBlank([
+                        'message' => 'payplug_sylius_payplug_plugin.secret_key.not_blank',
+                    ]),
+                    new IsPayPlugSecretKeyValid(),
+                    new IsCanSaveBancontactMethod(),
+                ],
+                'help' => $this->translator->trans('payplug_sylius_payplug_plugin.ui.retrieve_secret_key_in_api_configuration_portal'),
+                'help_html' => true,
+            ])
+            ->add(PayPlugGatewayFactory::ONE_CLICK, CheckboxType::class, [
```

Comment:
IMO, we don't have One Click on Bancontact, right ?

Reply:
They are removed in next PR